### DISCUSSION TITLE
Оффлайн-инсталлятор последней стабильной версии клиента

### DISCUSSION THREAD

**Discussion by birykvlad** [getlantern/lantern-forum-ru#2] [Q&A]
На сайте Lantern для Windows предлагается инсталлятор старой версии 6.10.5, которая после установки начинает обновляться до 7.2.х. Где можно скачать именно последнюю полную версию инсталлятора? На GitHub качается тоже старая версия.

**Comment by birykvlad** [comment]
Инсталлятор обновлен на GitHub, осталось обновить на сервере Amazon, ссылка на который представлена на официальном сайте для скачивания.
Спасибо ребятам за быструю реакцию, вопрос считаю решенным.

**Reply by Zonder3711** [reply]
Здравствуйте! Где на GitHub скачать обновленную версию? Спасибо.

**Reply by birykvlad** [reply]
Вот тут - https://github.com/getlantern/lantern
Для Windows прямая ссылка на стабильную версию будет такой - https://gitlab.com/getlantern/lantern-binaries-mirror/-/raw/master/lantern-installer.exe
